scripts/ci/_utils.sh

```diff
@@ -652,7 +652,7 @@ function run_flake8() {
             | tee -a "${OUTPUT_LOG}"
     else
         docker run "${AIRFLOW_CONTAINER_EXTRA_DOCKER_FLAGS[@]}" \
-            --entrypoint "/usr/local/bin/dumb-init"  \
+            --entrypoint "/usr/bin/dumb-init"  \
```

Comment:
What caused this path change?

Reply:
If dumb-init is installed by apt then it appears in this directory. More info: 
https://github.com/apache/airflow/pull/5659#discussion_r372017652

Reply:
I will replace it with the "init" field in docker compose.